bundles/AdminBundle/Controller/Admin/IndexController.php

```diff
@@ -247,6 +247,7 @@ protected function buildPimcoreSettings(Request $request, array &$templateParams
             'document_tree_paging_limit' => $config['documents']['tree_paging_limit'],
             'object_tree_paging_limit' => $config['objects']['tree_paging_limit'],
             'maxmind_geoip_installed' => (bool) $this->getParameter('pimcore.geoip.db_file'),
+            'hostname' => htmlentities(\Pimcore\Tool::getHostname(), ENT_QUOTES, 'UTF-8'),
```

Comment:
maybe here too? Not sure though if it really helps

Reply:
I'd assume Twig's `escape` would be also sufficient here, right?

Reply:
also the call to getHostname might be useful in some cases.